src/transformers/training_args.py

```diff
@@ -5,25 +5,15 @@
 from dataclasses import dataclass, field
 from typing import Any, Dict, Optional, Tuple
 
-from .file_utils import cached_property, is_torch_available, torch_required
+from .file_utils import cached_property, is_torch_available, is_tpu_available, torch_required
 
 
 if is_torch_available():
     import torch
 
-
-try:
+if is_tpu_available():
     import torch_xla.core.xla_model as xm
 
-    _has_tpu = True
```

Comment:
@julien-c I moved the `_has_tpu` logic to the `file_utils.py` since it's similar to `is_torch_available()` IMO.

Reply:
I think this is cool.

Reply:
Should we make explicit that it’s TPU available *in PyTorch*?

Reply:
Agree! Will change that to `is_torch_tpu_available()`